packages/react-core/src/components/DescriptionList/DescriptionListGroup.tsx

```diff
@@ -7,14 +7,24 @@ export interface DescriptionListGroupProps extends React.HTMLProps<HTMLDivElemen
   children?: any;
   /** Additional classes added to the DescriptionListGroup */
   className?: string;
+  /** Use Card component as a wrapper of DescriptionListGroup*/
+  isCard?: boolean;
 }
 
 export const DescriptionListGroup: React.FunctionComponent<DescriptionListGroupProps> = ({
   className,
   children,
+  isCard,
   ...props
 }: DescriptionListGroupProps) => (
-  <div className={css(styles.descriptionListGroup, className)} {...props}>
+  <div
+    className={css(
+      isCard && styles.card,
+      isCard ? 'pf-m-description-list__group' : styles.descriptionListGroup,
```

Comment:
I'm not sure if we want to use `pf-m-description-list__group` instead of `pf-c-description-list__group`.. I assume that the first option would be better, but couldn't find corresponding prop in styles. Should I add new prop there, or is it fine to use `pf-c-description-list__group` as usually? It seems that this does not change behavior of the Description List anyway.

Reply:
@mattnolting do you know which class is correct here?